projects/social-host/src/app/app-routing.module.ts

```diff
@@ -8,12 +8,12 @@ import { IoTComponent } from './controls/iot/iot.component';
 
 const routes: Routes = [
     {
-        path: 'applications/:appLookup/:appRoute/:projectLookup',
+        path: 'application/:appLookup/:appRoute/:projectLookup',
         component: ApplicationsComponent,
     },
-    { path: 'enterprises', component: EnterpriseComponent },
-    { path: 'projects/:projectLookup', component: ProjectsComponent },
-    { path: 'routes/:appRoute/:projectLookup', component: RoutesComponent },
+    { path: '', component: EnterpriseComponent },
+    { path: 'project/:projectLookup', component: ProjectsComponent },
+    { path: 'route/:appRoute/:projectLookup', component: RoutesComponent },
     { path: 'iiot', component: IoTComponent },
     { path: '', redirectTo: 'enterprises', pathMatch: 'full' },
```

Comment:
This should be a little bit different in here..  Line 14 for the EnterpriseComponent should move just below the iiot path.

For line 18, it should use angular's wildcard routing.  And would look something more like this:
`{ path: '**', redirectTo: '' },`

And this portion `, pathMatch: 'full'` would move into the EnterpriseComponent route.

The wildcard route should be the last in the list.